Sluggo-SwiftUI/Views/TicketEditDetail.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  TicketEditDetail.swift
+//  Sluggo-SwiftUI
+//
+//  Created by Troy Ebert on 2/4/23.
+//
+
+import Foundation
+import SwiftUI
+
+struct TicketEditDetail: View {
+    
+    @EnvironmentObject var identity: AppIdentity
+    @StateObject var alertContext = AlertContext()
+    
+    @Binding var ticket: TicketRecord
+    @Binding var showView: Bool
+
+    @State private var ticketTitle: String = ""
+    @State private var ticketUser: MemberRecord?
+    @State private var ticketStatus: StatusRecord?
+    @State private var ticketTags: [TagRecord] = []
+    @State private var ticketDueDate = Date()
+    @State private var dateToggle = false
+    @State private var ticketDescription: String = ""
```

Comment:
I feel like a good portion of the functions and variables of this view should be abstracted away to a viewModel. But I can do that after we merge this if you'd like.